apps/src/applab/sanitizeHtml.js

```diff
@@ -203,7 +203,8 @@ export default function sanitizeHtml(
       'value',
       'accept',
       'hidden',
-      'capture'
+      'capture',
+      'readonly'
```

Comment:
Awesome work! Is sanitizing the HTML needed since the DOM is cloned and transformed back and forth between the design, code, and run modes?

Reply:
I haven't traced through every call to this function, but I believe that's the general idea.